modular_skyrat/modules/primitive_fun/code/cutting_board.dm

```diff
@@ -0,0 +1,125 @@
+#define GET_RECIPE(input_thing) LAZYACCESS(processor_inputs[/obj/machinery/processor], input_thing.type)
+
+/obj/item/cutting_board
+	name = "cutting board"
+	desc = "Processing food before electricity was cool, because you can just do your regular cutting on the table next to this right?"
+	icon = 'modular_skyrat/modules/primitive_fun/icons/cooking_structures.dmi'
+	icon_state = "cutting_board"
+	force = 5
+	throwforce = 7 //Imagine someone just throws the entire fucking cutting board at you
+	w_class = WEIGHT_CLASS_NORMAL
+	pass_flags = PASSTABLE
+	layer = BELOW_OBJ_LAYER //So newly spawned food appears on top of the board rather than under it
+	///List containg list of possible inputs and resulting recipe items, taken from processor.dm and processor_recipes.dm
+	var/static/list/processor_inputs
+
+/obj/item/cutting_board/Initialize(mapload)
+	. = ..()
+	if(processor_inputs)
+		return
+	processor_inputs = list()
+	for(var/datum/food_processor_process/recipe as anything in subtypesof(/datum/food_processor_process)) //this is how tg food processors do it just in case this is digusting
```

Comment:
```suggestion
		return
	
	processor_inputs = list()
	for(var/datum/food_processor_process/recipe as anything in subtypesof(/datum/food_processor_process)) //this is how tg food processors do it just in case this is digusting
```